plugins/services/src/js/pages/task-details/TaskLogsTab.js

```diff
@@ -168,13 +178,11 @@ class TaskLogsTab extends mixin(StoreMixin) {
     const {task} = this.props;
     const {subscriptionID} = this.state;
 
-    // Stop tailing before fetching preiovus logs
-    SystemLogStore.stopTailing(this.state.subscriptionID);
-    // Fetch two pages previous log entries to gain more leverage to explore
+    // Fetch a full page previous log entries to gain more leverage to explore
```

Comment:
Yes, it probably shouldn't have been committed like that (the doubling), I guess I was testing out the size of the page and landed at 400, so I updated it to that.